common-client/src/main/scala/org/genivi/sota/client/DeviceRegistryClient.scala

```diff
@@ -91,6 +91,13 @@ class DeviceRegistryClient(baseUri: Uri, devicesUri: Uri, deviceGroupsUri: Uri,
     execHttp[NoContent](HttpRequest(method = PUT, uri = baseUri.withPath(mydeviceUri.path / device.show / "packages"),
       entity = HttpEntity(ContentTypes.`application/json`, packages.asJson.noSpaces)))
 
+  override def affectedDevices(namespace: Namespace, packageIds: Set[PackageId])
+                              (implicit ec: ExecutionContext): Future[Map[Uuid, Set[PackageId]]] = {
+    execHttp[Map[Uuid, Set[PackageId]]](HttpRequest(method = POST,
+      uri = baseUri.withPath(mydeviceUri.path / "affected"),
```

Comment:
This shouldn't be `mydeviceUri` since it should be "/api/v1/packages" I'm wondering if the config options are making things that much clearer...